frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -126,7 +127,7 @@ const Campaigns = () => {
       width: 'xs center',
     },
     {
-      name: 'Export',
+      name: '',
```

Comment:
`name` is used as the key for `td` and `th` in `renderCampaignList` and `renderRow`. Since they're both empty strings now, React will complain about duplicated keys. Perhaps use the index value as the key instead.

![image](https://user-images.githubusercontent.com/3666479/100118333-b1615580-2eb0-11eb-9b3f-691f9b682c0b.png)

Reply:
The key is already the index.  Thanks for raising, will look into it
```
<tbody>{campaignsDisplayed.map(renderRow)}</tbody>
```
```
  function renderRow(campaign: Campaign, key: number) {
    return (
      <tr key={key} onClick={() => history.push(`/campaigns/${campaign.id}`)}>
...
```

Reply:
oh nvm i was debugging the wrong part, youre right